src/pages/console/data/content-management.mdx

```diff
@@ -78,11 +78,11 @@ To use the data manager, you can [adjust your GraphQL API settings.](/console/da
 ### Null required fields
 When using the data manager, you may encounter the following error message:
 
-<Callout info>
+```console
 
-"Field {{fieldName}} cannot be set to null since its a required field."
+Field {{fieldName}} cannot be set to null since its a required field.
```

Comment:
Should be `it's`. If this is the actual error message, we should fix it there as well.

Reply:
looks like we need to change this on studio app too. since error was copy pasted